Product/pom.xml

```diff
@@ -59,7 +59,7 @@
             </dependency>
 
             <!-- CXF -->
-            <dependency>
+            <!--<dependency>
```

Comment:
Let's go ahead and delete the commented out dependencies unless you think they will need to be added back in.